zetaclient/chains/solana/signer/signer.go

```diff
@@ -167,6 +194,19 @@ func (signer *Signer) GetGatewayAddress() string {
 	return signer.gatewayID.String()
 }
 
+// SetRelayerBalanceMetrics sets the relayer balance metrics
+func (signer *Signer) SetRelayerBalanceMetrics(ctx context.Context) {
+	if signer.HasRelayerKey() {
+		result, err := signer.client.GetBalance(ctx, signer.relayerKey.PublicKey(), rpc.CommitmentFinalized)
+		if err != nil {
+			signer.Logger().Std.Error().Err(err).Msg("GetBalance error")
+			return
+		}
+		solBalance := float64(result.Value) / float64(solana.LAMPORTS_PER_SOL)
+		metrics.RelayerKeyBalance.WithLabelValues(signer.Chain().Name).Set(solBalance)
+	}
```

Comment:
nit: ideally we'd maintain lower identation so the code is more readable.
```suggestion
	if !signer.HasRelayerKey() {
		return
	}
	
	result, err := signer.client.GetBalance(ctx, signer.relayerKey.PublicKey(), rpc.CommitmentFinalized)
	if err != nil {
		signer.Logger().Std.Error().Err(err).Msg("GetBalance error")
		return
	}
	solBalance := float64(result.Value) / float64(solana.LAMPORTS_PER_SOL)
	metrics.RelayerKeyBalance.WithLabelValues(signer.Chain().Name).Set(solBalance)
	}
```

Reply:
https://github.com/zeta-chain/node/pull/2673/commits/3a415d21bc20cf6e6ce8d79cfc0ae1249d1a3201